### PR TITLE
Adapt to new processing config structure

### PR DESCRIPTION
**Pull request by aerlaut**
# Background
#### Link to issue 

https://biomage.atlassian.net/browse/BIOMAGE-671

#### Link to staging deployment URL 

https://ui-xavier-ui163.scp-staging.biomage.net

#### Links to any Pull Requests related to this

N.A.

#### Anything else the reviewers should know about the changes here

# Changes
### Code changes

# Definition of DONE
Your changes will be ready for merging after each of the steps below have been completed:

### Testing
- [ ] Unit tests written
- [X] Tested locally with Inframock
- [X] Deployed to staging

To set up easy local testing with inframock, follow the instructions here: https://github.com/biomage-ltd/inframock
To deploy to the staging environment, follow the instructions here: https://github.com/biomage-ltd/biomage-utils

### Documentation updates
Is all relevant documentation updated to reflect the proposed changes in this PR?

- [ ] Relevant Github READMEs updated
- [ ] Relevant wiki pages created/updated

### Approvers
- [ ] Approved by a member of the core engineering team
- [ ] (UX changes) Approved by vickymorrison (this is her username, tag her if you need approval)

### Just before merging:
- [x] After the PR is approved, the `unstage` script in here: https://github.com/biomage-ltd/biomage-utils is executed. This script cleans up your deployment to staging

### Optional
- [ ] Photo of a cute animal attached to this PR

### CODE REVIEW

**Comment by aerlaut** [comment]
The API part of the ticket is fixed together with https://github.com/biomage-ltd/api/commit/897a7ce3fffec4af46d9b1ddeeab53b7aa9046c9

File: src/redux/reducers/experimentSettings/initialState.js [hunk 1]
```diff
@@ -28,19 +28,22 @@ export default {
...
       filterSettings: {
         regressionType: 'gam',
-        smoothing: 13,
-        upperCutoff: 4.8,
-        lowerCutoff: 2.1,
-        stringency: 2.1,
-        binStep: 0.05,
+        regressionTypeSettings: {
+          gam: {
+            'p.level': 9.090083e-05,
```
Comment (aerlaut) [inline src/redux/reducers/experimentSettings/initialState.js]: Need to clarify this initial value with the bioinformaticians

**Comment by cosa65** [comment]
Could you add a staging?

**Comment by cosa65** [comment]
This ticket involves also adapting the processing config validation on the api side too

**Review by xverges — APPROVED** [review]
Sorry for taking so long to review: now some merge conflicts have appeared (and potentially with other active PRs) :-(

It does not look to me that the auto radio button works properly, but I'll work on that as a part of my ticket.

**Comment by aerlaut** [comment]
> Sorry for taking so long to review: now some merge conflicts have appeared (and potentially with other active PRs) :-(
> 
> It does not look to me that the auto radio button works properly, but I'll work on that as a part of my ticket.

Right now there is no functionality for the auto radio button, it just updates the redux store.

**Pull request merged**